src/components/DatasetItem.vue

```diff
@@ -7,6 +7,23 @@
       </dataset-info>
     </el-dialog>
 
+    <div class="opt-image">
+      <router-link :to="opticalImageAlignmentHref" v-if="opticalImageSmall && haveEditAccess && dataset.status != 'STARTED'" title="Edit Optical Image">
+        <div class="edit-opt-image">
+            <img class="opt-image-thumbnail" :src="opticalImageSmall" width="100px" height="100px" alt="edit_optical_image"/>
+        </div>
+      </router-link>
+      <router-link :to="opticalImageAlignmentHref" v-else-if="haveEditAccess && dataset.status != 'STARTED'" title="Add Optical Image">
+        <div class="no-opt-image" >
+            <img src="../assets/empty_rect_100_100.png" width="100px" height="100px" alt="edit_optical_image"/>
+        </div>
+      </router-link>
+      <div class="edit-opt-image-guest" v-else-if="dataset.status != 'STARTED'">
+        <img v-if="opticalImageSmall" :src="opticalImageSmall" width="100px" height="100px" alt="optical_image"/>
+        <img v-else src="../assets/empty_rect_100_100.png" width="100px" height="100px" alt="optical_image"/>
```

Comment:
I think using an empty image is not justified here :) empty div would work just fine

Reply:
oh yes, that's true, thanks! :)